tests/rptest/tests/tx_admin_api_test.py

```diff
@@ -0,0 +1,233 @@
+# Copyright 2021 Vectorized, Inc.
+#
+# Use of this software is governed by the Business Source License
+# included in the file licenses/BSL.md
+#
+# As of the Change Date specified in that file, in accordance with
+# the Business Source License, use of this software will be governed
+# by the Apache License, Version 2.0
+
+from ducktape.mark.resource import cluster
+from rptest.clients.types import TopicSpec
+from rptest.clients.kafka_cat import KafkaCat
+from ducktape.utils.util import wait_until
+from rptest.services.admin import Admin
+
+import confluent_kafka as ck
+
+import random
+import time
+
+from rptest.tests.redpanda_test import RedpandaTest
+
+
+class TxAdminTest(RedpandaTest):
+    topics = (TopicSpec(name="tx_test",
+                        partition_count=3,
+                        replication_factor=3), )
+
+    def __init__(self, test_context):
+        super(TxAdminTest,
+              self).__init__(test_context=test_context,
+                             num_brokers=3,
+                             extra_rp_conf={
+                                 "enable_idempotence": True,
+                                 "enable_transactions": True,
+                                 "tx_timeout_delay_ms": 10000000,
+                                 "abort_timed_out_transactions_interval_ms":
+                                 10000000,
+                                 'enable_leader_balancer': False
+                             })
+
+        self.admin = Admin(self.redpanda)
+
+    def transfer_leadership(self, topic_name, new_leader=None):
```

Comment:
in addition to what @rystsov had to say about `transfer_leadership`, I think that we already have a leadership transfer utility available in duck tape. we should prefer to re-use / generalize it as much as possible.